services/frontend-service/src/ui/utils/Links.tsx

```diff
@@ -147,6 +154,22 @@ export const DisplayManifestLink: React.FC<{ displayString: string; app: string;
     return null;
 };
 
+export const DisplayCommitHistoryLink: React.FC<{ displayString: string; commitId: string }> = (
```

Comment:
Since this is a PR that changes the UI:
please take a screenshot of your changes, and paste it into the PR description here on GitHub.

Reply:
And please describe (in the PR description) what this change does a little more. e.g
"Introduces a new link on the release dialog that points to the new commit page"